app/controllers/api/v1/base_controller.rb

```diff
@@ -1,6 +1,5 @@
 class Api::V1::BaseController < ApplicationController
   
-  # Generic API stuff here
   before_filter :skip_trackable
```

Comment:
before_filter will be deprecated in Rails 5, https://stackoverflow.com/questions/16519828/rails-4-before-filter-vs-before-action, so should this changes be made?

Reply:
It was indeed a mistake, I forgot to rebase.

Reply:
okay 👍🏼